e2e/testdata/fn-eval/include-metadata/.expected/config.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Google LLC
```

Comment:
Pl. rename this test to `include-metaresources` instead of `include-metadata`.

Reply:
done